config/routes.rb

```diff
@@ -7,4 +7,6 @@
   post '/login', to: 'sessions#create'
 
   get '/dashboard', to: 'user/dashboard#index'
+
+  resources :friends, only: [:create]
```

Comment:
This looks good. If we only have a route it may be more readable to spell it out. Something to consider later on if we don't add more routes